cmd/snap-bootstrap/README.md

```diff
@@ -18,60 +18,60 @@ _snap-bootstrap_ is the main executable that is run during the initramfs stage o
 1. After having identified which partition is ubuntu-seed, we mount it at /run/mnt/ubuntu-seed.
 1. Next, we will load the "recovery system seed", which is the set of snaps associated with this recovery system, this includes the base snap, the kernel snap, the snapd snap and the gadget snap. These snaps are verified to match their assertions via hashing.
 1. Next we do another measurement to the TPM (if available) of the model assertion from the recovery system we loaded.
 1. After having verified that the recovery system seed snaps are valid and that the model assertion is correct, we will then mount these snaps at /run/mnt/base, /run/mnt/kernel, and /run/mnt/snapd (the gadget is not mounted at this time).
 1. Next, we create a tmpfs mount at /run/mnt/data, which will be the root filesystem we pivot_root into at the end of the initramfs. 
-1. Next, we will "configure" the target system root filesystem using the gadget snap itself, this will handle things like "early snap config" and cloud-init config, etc. that needs to be applied before we fully boot to userspace.
+1. Next, we will "configure" the target system root filesystem using the gadget snap itself, this will handle things like "early snap config" and cloud-init config, etc. that need to be applied before we fully boot to userspace.
 1. Next, we will write out a modeenv file to the root filesystem based on the model assertion and the recovery system seed snaps that will be read by snapd in userspace when we get there.
-1. Finally, the last step of all modes is to expose the boot flags that were put into the boot environment for userspace to measure. This is done via `boot.InitramfsExposeBootFlagsForSystem`
+1. Finally, the last step of all modes is to expose the boot flags that were put into the boot environment for userspace to measure. Currently there is only one boot flag used for allowing factory specific behavior in the install-device hook during install mode that is not to be executed if the device is re-installed in the field and enters install mode again. The boot flag is set by a bootloader environment variable, which is then put into a file in /run for userspace to measure. See https://ubuntu.com/core/docs/uc20/installation-process for full details of how this can be used from a image building standpoint and see the implementation of `boot.InitramfsExposeBootFlagsForSystem` for how this works at a low level as a snapd/Ubuntu Core developer.
```

Comment:
```suggestion
1. Finally, the last step of all modes is to expose any boot flags. There is currently only one boot flag and this is used during install mode to allow factory-specific behavior in the install-device hook, stopping re-execution if the device is re-installed in the field and re-enters install mode again. A boot flag is set by a bootloader environment variable which is then put into a file in /run for userspace to measure. See https://ubuntu.com/core/docs/uc20/installation-process for full details of how this can be used from an image building standpoint, and see the implementation of `boot.InitramfsExposeBootFlagsForSystem` for how this works at a low-level for a snapd/Ubuntu Core developer.
```